src/app.js

```diff
@@ -428,6 +428,7 @@ class Application {
     }
 
     getCardUsageCount(searchedCard) {
+        // TODO: cleanup
```

Comment:
This isn't about occupancy, what is to cleanup here?

Reply:
For now forget about it. I was thinking about small refactoring, maybe in the future